include/nuttx/fs/uio.h

```diff
@@ -45,6 +45,7 @@ struct uio
 {
   FAR const struct iovec *uio_iov;
   int uio_iovcnt;
+  size_t uio_offset_in_iov; /* offset in uio_iov[0].iov_base */
```

Comment:
uio_offset

Reply:
i'd like to reserve the name "uio_offset" for file offsets as it's what's meant by the name in BSDs.